doc/index.rst

```diff
@@ -5,7 +5,35 @@ Welcome to desiutil's documentation!
 Introduction
 ============
 
-This is the documentation for desiutil.
+desiutil is a set of low-level utilities used by all DESI_ packages.
+
+.. _DESI: https://desi.lbl.gov
+
+Dependencies
+============
+
+Required Dependencies
++++++++++++++++++++++
+
+These packages must be installed for desiutil to work properly:
+
+* `pyyaml <http://pyyaml.org/>`_
+* `astropy <http://www.astropy.org/>`_
+
+  - This implies a dependency on `NumPy <http://www.numpy.org>`_
+
+Optional Dependencies
++++++++++++++++++++++
+
+If you want to use the plotting utilities in :mod:`desiutil.plots`, you will
+need:
+
+* `matplotlib <http://matplotlib.org/>`_
+* `healpy <https://healpy.readthedocs.io/en/latest/>`_
```

Comment:
Yes, you're right.  But healpix-deniers can still make pretty plots without installing healpy if they insist.